services/movie.py

```diff
@@ -1,14 +1,18 @@
 from django.db.models import QuerySet
-
+from django.db import transaction
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Ensure that the type annotations for `genres_ids` and `actors_ids` are correct. If you are using Python 3.9 or later, `list[int]` is fine. Otherwise, consider using `List[int]` from `typing` for compatibility.